library/oeUI/src/OemrUI.php

```diff
@@ -0,0 +1,290 @@
+    *
+    * @param int|bool  $display_help_icon
+    *
+    * @return string $help_icon that will output the help icon html string
+    *
+    */
+    
+    private function helpIcon($display_help_icon = '')
+    {
+        $display_help_icon = $this->display_help_icon;
+        if ($display_help_icon) {
+            if ($_SESSION ['language_direction'] == 'ltr') {
+                $help_icon_title = "To enable help - Go to the User Name on top right > Settings > Features > Enable Help Modal";
+            } elseif ($_SESSION ['language_direction'] == 'rtl') {
+                $help_icon_title = "To enable help - Go to the User Name on top left > Settings > Features > Enable Help Modal";
+            }
+            if ($GLOBALS['enable_help'] == 1) {
+                $help_icon = '<a class="oe-pull-away oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#676666" title="' . xla("Click to view Help") . '"><i class="fa fa-question-circle" aria-hidden="true"></i></a>';
+            } elseif ($GLOBALS['enable_help'] == 2) {
+                $help_icon = '<a class="oe-pull-away oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#DCD6D0 !Important" title="' . xla($help_icon_title) . '"><i class="fa fa-question-circle" aria-hidden="true"></i></a>';
+            } elseif ($GLOBALS['enable_help'] == 0) {
+                 $help_icon = '';
+            }
+        }
+        return $help_icon;
+    }
+       
+    /**
+    * Used to output the help modal html along with the jQuery to make it work.
+    *
+    * @param string $help_file - name of the help file to be displayed, must exists in Documentation/help_files
+    * will echo the entire html string of the help modal and the jQuery, needs to be used as the first line after the container div
+    *
+    * @return void
+    *
+    */
+    public function helpFileModal()
+    {
+        $help_file = $this->help_file;
+        $close = xla("Close");
+        $print = xla("Print");
+        if ($help_file) {
+            $help_file = attr($help_file);
+            $help_file = $GLOBALS['webroot']."/Documentation/help_files/$help_file";
+            $modal_body = "<iframe src=\"$help_file\" id='targetiframe' style='height:100%; width:100%; overflow-x: hidden; border:none'
+                                allowtransparency='true'></iframe>";
+        } else {
+            $modal_body = "<h3> <i class='fa fa-exclamation-triangle  oe-text-red' aria-hidden='true'></i> " . xlt("Check if a help file exists for this page in") . " " . text("Documentation/help_files") . ".<br><br>" . xlt("Then pass it's name as an argument in") . " " . text("function helpFileModal"). ".<br><br>" . xlt("If the help file does not exist create one and place it in") . " " . text("Documentation/help_files") . ".<br>" . "</h3>";
```

Comment:
$modal_body = "<h3> <i class='fa fa-exclamation-triangle  oe-text-red' aria-hidden='true'></i> " . xlt("Check if a help file exists for this page in Documentation/help_files") . ".<br><br>" . xlt("Then pass it's name as an argument in function helpFileModal"). ".<br><br>" . xlt("If the help file does not exist create one and place it in Documentation/help_files") . ".<br>" . "</h3>";